src/collar/oc_bookmarks.lsl

```diff
@@ -1,523 +1,606 @@
-        llMessageLinked(LINK_DIALOG,NOTIFY,"0"+"The maximum number 45 bookmars is already reached.",kID);
-        return;
-    }
-    llMessageLinked(LINK_SAVE, LM_SETTING_SAVE, g_sSettingToken + sMessage + "=" + sLoc, "");
-    g_lVolatile_Destinations += sMessage;
-    g_lVolatile_Slurls += sLoc;
-    llMessageLinked(LINK_DIALOG,NOTIFY,"0"+"Added destination " + sMessage + " with a location of: " + sLoc,kID);
+AddDestination(string sMessage, string sLoc, key kID) {
+  if (llGetListLength(g_lVolatileDestinations) + llGetListLength(g_lDestinations) >= 45) {
+    llMessageLinked(LINK_DIALOG, NOTIFY, "0" + "The maximum number 45 bookmars is already reached.", kID);
+    return;
+  }
+
+  llMessageLinked(LINK_SAVE, LM_SETTING_SAVE, g_sSettingToken + sMessage + "=" + sLoc, "");
+  g_lVolatileDestinations += sMessage;
+  g_lVolatileSLUrls += sLoc;
+  llMessageLinked(LINK_DIALOG, NOTIFY, "0" + "Added destination " + sMessage + " with a location of: " + sLoc, kID);
 }
 
 string FormatRegionName() {
-    //Formatting to regionname(x,x,x)
-    string region = llGetRegionName();
-    vector pos = llGetPos();
-    string posx = (string)llRound(pos.x);
-    string posy = (string)llRound(pos.y);
-    string posz = (string)llRound(pos.z);
-    return (region + "(" + posx + "," + posy + "," + posz + ")");
+  // Formatting to regionname(x,x,x)
+  string sRegion = llGetRegionName();
+  vector vPos = llGetPos();
+  string sPosX = (string)llRound(pos.x);
+  string sPosY = (string)llRound(pos.y);
+  string sPosZ = (string)llRound(pos.z);
+  return (sRegion + "(" + sPosX + "," + sPosY + "," + sPosZ + ")");
 }
 
-string convertSlurl(string sStr) {  //convert the slurl http strings to region (xxx,xxx,xxx)
-    sStr = llStringTrim(llUnescapeURL(sStr), STRING_TRIM);
-    string sIndex = "http:";
-    list lPieces =  llParseStringKeepNulls(sStr, ["/"], []);
-    integer iHttploc = 0;
-    string sStringToKeep;
-    integer iHttpInString = llSubStringIndex(llList2String(lPieces, 0), sIndex);
-    if(iHttpInString > 0)
-        sStringToKeep = llGetSubString(llList2String(lPieces, 0), 0, iHttpInString - 1);
-    if(llGetListLength(lPieces) == 8) {
-        string sRegion = llList2String(lPieces, iHttploc + 4);
-        string sLocationx = llList2String(lPieces, iHttploc + 5);
-        string sLocationy = llList2String(lPieces, iHttploc + 6);
-        string sLocationz = llList2String(lPieces, iHttploc + 7);
-        sStr = sStringToKeep + sRegion + "(" + sLocationx + "," + sLocationy + "," + sLocationz + ")";
-        //Debug("Converted slurl, sending for processing..." + sStr);
-        return sStr; //successful conversion, send converted string assembly
-    }
-    //Debug("No slurl detected, sending for processing anyways :" + sStr);
-    return sStr; //failed conversion, send raw string assuming there's no Slurls
+string ConvertSLUrl(string sStr) { // convert the slurl http strings to region (xxx,xxx,xxx)
```

Comment:
It appears the indentation in this function was also removed and removed with fewer spaces as well as functions above it.